fortran/micm_core.F90

```diff
@@ -3,12 +3,13 @@ module micm_core
    use iso_c_binding, only: c_ptr, c_char, c_int, c_double, c_null_char, c_size_t, c_f_pointer
    implicit none
 
-   public :: micm_t
+   public :: micm_t, Mapping
```

Comment:
I think we've been using the style `mapping_t` for derived types in Fortran

Reply:
ah, yes. fixing that now